src/test/run-make-fulldeps/coverage-reports/Makefile

```diff
@@ -98,28 +98,29 @@ endif
 			)
 
 	# Run it through rustdoc as well to cover doctests
-	LLVM_PROFILE_FILE="$(TMPDIR)"/$@-%p.profraw \
+	LLVM_PROFILE_FILE="$(TMPDIR)"/$@-%p-%m.profraw \
```

Comment:
I _think_ this will help, assuming our theory is correct that there could be PID-reuse on Windows. As I understand it, every doc test produces a new, program (new `main()` and unique source) so the binary signature (`%m`) should be different for each one.

In theory, adding the PID `%p` value is probably superfluous, because `%m` _should_ be unique enough, but maybe keeping `%p` is an added safeguard to ensuring we don't have filename collisions, and there's probably harm keeping both.